### PR TITLE
Back and forth clusters buttons

### LINKED ISSUE #194: No previous/next buttons to fly to filtered clusters

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Fix #194

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Here is a template for new release sections
 - clicking on village button will randomly select a cluster in a random state (#175)
 - clicking on state button will randomly select a state
 - ability to download the information about settlements clusters in a csv file (#156)
+- add next and previous buttons (#194)
```
Comment (Bachibouzouk) [inline CHANGELOG.md]:
```suggestion
- add next and previous buttons (#205)
```

In changelog we refer to PR numbers (you can say it fixed Issue #194)

**Review by Bachibouzouk — COMMENTED** [review]
There are no buttons, did you forget to push the changes in the html file?

File: CHANGELOG.md (hunk 1 above)
Comment (christian-rli) [reply CHANGELOG.md]: ✓

**Comment by christian-rli** [comment]
Latest commit deals with situation when unable to load geojson. Buttons become inert. Map continues to work otherwise. Can we merge into dev @Bachibouzouk ?

**Pull request merged**